app/jenkins-for-jira-ui/src/components/InProductHelpDrawer/InProductHelpAction.tsx

```diff
@@ -20,24 +31,60 @@ export enum InProductHelpActionType {
 type InProductHelpActionProps = {
 	label: string,
 	type: InProductHelpActionType,
-	appearance?: InProductHelpActionButtonAppearance
+	appearance?: InProductHelpActionButtonAppearance,
+	screenName?: string
+};
+
+const analyticsClient = new AnalyticsClient();
+
+const iphClickSource = (screenName?: string): string => {
+	switch (screenName) {
+		case SET_UP_GUIDE_SCREEN_NAME:
+			return AnalyticsScreenEventsEnum.ServerManagementScreenName;
+			break;
```

Comment:
dont need any of the `breaks` since your returning.

Reply:
Oh right. Had those in there originally as placeholders and forgot to clear them out!